src/openforms/emails/templates/emails/templatetags/product_information.txt

```diff
@@ -0,0 +1 @@
+{% if product and product.information %}{{ product.information|safe }}{% endif %}
```

Comment:
This will print out HTML in a txt template, no?

I would see if we can re-use the e-mail HTML-to-text functionality in the form of a template filter to convert this to a plain text format.

Reply:
Yes, but the HTML gets cleaned up by the HTML-to-text, just like the other rich textfields.

Reply:
right, I had the flow inverted in my head, thanks for the clarification